### PR TITLE
Fix the issues in branding preview section

### PR DESCRIPTION
**Pull request by DilshanSenarath**
### Purpose
- Remove customization and support mail from branding section
- Updated the custom layout section to accommodate support for IS
- Remove `PREMIUM` label from custom layout card

### Related Issues
- https://github.com/wso2/product-is/issues/16743
- https://github.com/wso2/product-is/issues/16747

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/extensions/components/branding/components/preview/branding-preference-preview.tsx [hunk 1]
```diff
@@ -334,51 +345,81 @@ export const BrandingPreferencePreview: FunctionComponent<BrandingPreferencePrev
...
+                                                                <Trans
+                                                                    key={ 1 }
+                                                                    i18nKey={ "extensions:develop.branding."
+                                                                        + "tabs.preview.errors.layout."
+                                                                        + "notFoundWithSupport.description" }
+                                                                    tOptions={ {
+                                                                        supportEmail
+                                                                    } }
+                                                                >
+                                                                    Need a fully customized layout for your
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/components/branding/components/preview/branding-preference-preview.tsx:394]: Don't we need to i18n these strings?

File: apps/console/src/extensions/components/branding/components/preview/branding-preference-preview.tsx [hunk 2]
```diff
@@ -36,6 +44,8 @@ import { EmailTemplateScreenSkeleton } from "./email-template-screen-skeleton";
 import { LoginScreenSkeleton } from "./login-screen-skeleton";
 import { MyAccountScreenSkeleton } from "./my-account-screen-skeleton";
 import { AppState } from "../../../../../features/core/store";
+import { ReactComponent as CustomLayoutSuccessImg } from
+    "../../../../../themes/wso2is/assets/images/branding/custom-layout-success.svg";
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/components/branding/components/preview/branding-preference-preview.tsx:48]: We should not hardcode the theme names, instead we should import from the themes module

**Review by pavinduLakshan — APPROVED** [review]
Let's fix the comments in a follow-up pr if this pr is urgent

File: apps/console/src/extensions/components/branding/components/preview/branding-preference-preview.tsx (hunk 1 above)
Comment (DilshanSenarath) [reply apps/console/src/extensions/components/branding/components/preview/branding-preference-preview.tsx:394]: Already added the i18n. This is the `Trans` component (You can see the provided `i18nKey` as a prop) and the code is an existing one.

File: apps/console/src/extensions/components/branding/components/preview/branding-preference-preview.tsx (hunk 2 above)
Comment (DilshanSenarath) [reply apps/console/src/extensions/components/branding/components/preview/branding-preference-preview.tsx:48]: Currently I couldn't found a way to include the images without hardcoding the theme. There are some other places as well we need to fix this. https://github.com/wso2/identity-apps/blob/8e833d1848e906f28aae8265bc606156f1be1454/apps/console/src/features/core/components/header.tsx#L48-L56

**Pull request merged**